zilencer/management/commands/populate_db.py

```diff
@@ -899,6 +898,8 @@ def handle(self, **options: Any) -> None:
             mark_all_messages_as_read()
             self.stdout.write("Successfully populated test database.\n")
 
+        push_notifications_logger.disabled = False
```

Comment:
I think probably the important part is just moving this out of the top level, so that importing this file doesn't result in that change occurring.